### ISSUE TITLE
Glider found

### ISSUE THREAD

**Issue by wfowler1** [chrisevans9629/GameOfLife#24]
I've found a simple glider for your B4/S56 3D ruleset. It has a 6-cell setup, and moves up continuously on the Y-axis.

0,0,0
0,0,1
1,0,0
1,0,1
1,2,2
2,2,1

Demo: https://github.com/wfowler1/GameOfLife/releases/tag/v1.0.4
Run the program, go to Presets -> 3D Demos -> Evans Glider

**Comment by AutumnEvans418** [comment]
That is awesome that you found that! The 4D configurations are also really cool! I appreciate you sharing that and that you found my blog useful! I'll have to update it 👍 :)

**Comment by wfowler1** [comment]
Yeah, I just noticed it while playing around with the ruleset with very low initial populations. Starting out with a single 2x2x1 block makes some really cool patterns. While exploding into nonsense.

But I figured you probably would like to hear about a glider, since your blog mentioned you hadn't found one. It seemed like you'd like to know since you documented the oscillators.
